nav2_gps_waypoint_follower_demo/config/nav2_no_map_params.yaml

```diff
@@ -196,10 +195,11 @@ local_costmap:
           raytrace_min_range: 0.0
           obstacle_max_range: 2.5
           obstacle_min_range: 0.0
+      static_layer:
```

Comment:
I don't think this is used in the configuration

Reply:
done in https://github.com/ros-navigation/navigation2_tutorials/pull/115/commits/fd521ea5a04418d8a265ed30ecb6cd07d40dcc01